src/ImageSharp/ImageSharp.csproj

```diff
@@ -29,7 +29,7 @@
     <DebugType Condition="$(codecov) == ''">portable</DebugType>
     <DebugSymbols>True</DebugSymbols>
     <Features>IOperation</Features>
-    <LangVersion>7.3</LangVersion>
+    <LangVersion>Latest</LangVersion>
```

Comment:
I wonder how will AppVeyor react if we undo this. This is the only change that has something to do with the build.

Reply:
Looks like other PR's also have build issues?

Reply:
Use `Previous Visual Studio 2017`image in appveyor.yml. 

https://ci.appveyor.com/project/six-labors/imagesharp/build/1.0.0-PullRequest00649000167

Can someone raise an issue with them please.

Reply:
NM. Managed to do it . 

https://github.com/appveyor/ci/issues/2519